core/src/main/java/org/elasticsearch/common/io/stream/StreamOutput.java

```diff
@@ -316,6 +316,18 @@ public void writeStringArrayNullable(@Nullable String[] array) throws IOExceptio
         }
     }
 
+    /**
+     * Writes a string array, for nullable string, writes it as 0 (empty string).
+     */
```

Comment:
+1 should this change be part of this PR? Looks like this is not used anywhere else.